fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayTestTag.kt

```diff
@@ -0,0 +1,27 @@
+package org.mozilla.fenix.tabstray
+
+internal object TabsTrayTestTag {
```

Comment:
I wish Compose offered a better way of interacting with view objects so that we didn't have to do this, but I think this makes more sense than replicating these strings across the Composables and the test code.